docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
```

Comment:
We have jquery loaded so should just use it here: ``$("#togglesidebar").keydown(handleEnterKey)``